CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to the versioning scheme outlined in the [README.md](README.md).
 
-## [Unreleased]
+## [3.1.0.0.8-rc1]
```

Comment:
It feels a little weird to to use the `-rc1` in the changelogs. Should it just be `3.1.0.0.8`?

Reply:
i can make that change, i just did it for correctness on this rc branch (but this shouldn't be merged back to develop/master)

Reply:
985436ef7c18aca59377cf5ac883e1eadf506bb3